docs/markdown/recursor/settings.md

```diff
@@ -23,6 +23,7 @@ queries needed.
 ## `allow-from`
 * IP ranges, separated by commas
 * Default: 10.0.0.0/8, 172.16.0.0/12, 192.168.0.0/16
+* Available since: 2.9.21
```

Comment:
The reasoning behind this was simple: some options already had these "ancient references" in place, so - for completeness' sake - I kept on with'em ;-)